src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+                    {
+                        switch (args[i])
+                        {
+                            case "info":
+                                executor.usage();
+                                end = true;
+                                break;
+                            case "help":
+                                Console.WriteLine("Usage: build [command]:");
+                                ++i;
+                                if (i < argsLength)
+                                {
+                                    end = true;
+                                    if (args[i].Equals("properties"))
+                                    {
+                                        jsonSetup.ProvideHelpProperties();
+                                    }
+                                    else if (args[i].Equals("commands"))
+                                    {
+                                        jsonSetup.ProvideHelpInstructions(os);
+                                    }
+                                }
+                                if (!end)
+                                {
+                                    jsonSetup.help(os);
+                                    end = true;
+                                }
+                                break;
+                            default:
+                                command = args[i];
+                                break;
+                        }
+                    }
+                    
+                    if(!end)
+                    {
+                        if (jsonSetup.commands.ContainsKey(command))
+                        {
+                            Console.WriteLine(jsonSetup.buildCommand(jsonSetup.commands[command], os));
+                        }
+                        else
+                        {
+                            Console.WriteLine(string.Format("Error: Command {0} not found.", command));
+                            jsonSetup.help(os);
+                        }
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("Error: Could not load Json configuration file.");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Error: Json configuration file should be provided.");
+            }
+        }
+    }
+
+    public class Setup
+    {
+        public Dictionary<string, Dictionary<string, string>> properties { get; set; }
+        public Dictionary<string, Command> commands { get; set; }
+        public Dictionary<string, Process> processes { get; set; }
+
+        public void ProvideHelpInstructions(string os)
+        {
+            Console.WriteLine("========Commands========");
+
+            foreach (KeyValuePair<string, Command> instr in commands)
+            {
+                Console.WriteLine(string.Format("* {0} - {1}", instr.Key, instr.Value.explanation));
```

Comment:
string.format is unnecessary, as you can pass a format string directly to Console.WriteLine.